reference/UiTPAS.v2.json

```diff
@@ -2420,125 +2542,764 @@
+                        "description2": "string",
+                        "pictures": [
+                          "string"
+                        ],
+                        "publicationPeriodBegin": "2019-08-24T14:15:22Z",
+                        "publicationPeriodEnd": "2019-08-24T14:15:22Z",
+                        "exchangePeriodBegin": "2019-08-24T14:15:22Z",
+                        "exchangePeriodEnd": "2019-08-24T14:15:22Z",
+                        "points": 0,
+                        "types": [
+                          "string"
+                        ],
+                        "forKids": true,
+                        "sport": true,
+                        "labels": "string",
+                        "moreInfoURL": "string",
+                        "cashInType": "PHYSICAL",
+                        "usesCodes": true,
+                        "ticketSaleDiscount": {
+                          "amount": 0,
+                          "type": "ABSOLUTE",
+                          "conditions": [
+                            {
+                              "type": "EVENT",
+                              "values": [
+                                "string"
+                              ]
+                            }
+                          ]
+                        }
+                      },
+                      "exchangeDate": "2019-08-24T14:15:22Z",
+                      "exchangeLink": "string",
+                      "exchangeInfo": {
+                        "exchangeText": "string",
+                        "exchangeLink": "string",
+                        "exchangeLinkLabel": "string"
+                      },
+                      "exchangeCode": "string"
+                    }
+                  }
+                }
+              }
+            }
+          }
+        },
+        "description": "Exchange passholder points for a reward.\n\nThe caller of this request must have `REWARDS_EXCHANGE` permission for the given organizer.",
+        "tags": [
+          "Rewards"
+        ],
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "type": "object",
+                "properties": {
+                  "uitpasNumber": {
+                    "type": "string"
+                  }
+                },
+                "required": [
+                  "uitpasNumber"
+                ]
+              },
+              "examples": {
+                "example-1": {
+                  "value": {
+                    "uitpasNumber": "string"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "security": [
+          {
+            "CLIENT_ACCESS_TOKEN": []
+          },
+          {
+            "USER_ACCESS_TOKEN": []
+          }
+        ]
+      },
+      "get": {
+        "summary": "Check reward exchange status for a passholder",
```

Comment:
I think this operation is still missing some things:

- It says `Check the reward exchange status for a passholder`. How do you specificy which passholder?
- The only response is 200 OK with no response body. I think this will probably return an `ExchangedReward` and/or a 4XX?